config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     secret: process.env.KEYCLOAK_SECRET
   },
   awsSqs: {
-    region: process.env.AWS_REGION,
+    region: process.env.AWS_REGION || 'eu-west-2',
```

Comment:
Does the region need to be hard coded here? I think it is already set in hof-services-config to be picked up in other environments.